### DISCUSSION TITLE
List Export Plugin use tutuorials.

### DISCUSSION THREAD

**Discussion by V0l9emar** [DiscipleTools/disciple-tools-list-exports#21] [Q&A]
Hello to everyone! Would you be able to explain to me how I can use this plugin? I installed it but can't find how to show it on the Disciples Tools main page...

**Comment by corsacca** [comment]
Hey @V0l9emar, welcome.
The plugin needs to be installed and **activated**.
The functionality will then appear on each list page like this:
![image](https://github.com/DiscipleTools/disciple-tools-list-exports/assets/24901539/5a9947c8-39f2-4bfe-bd45-524f68973cdd)
